mindsdb/integrations/handlers/anthropic_handler/README.md

```diff
@@ -1,11 +1,23 @@
 # Anthropic Handler
-Anthropic ML handler integrates Anthropic LLMs with MindsDB. You can train your existing text data with Anthropic LLM
+The Anthropic ML handler integrates Anthropic LLMs with MindsDB. You can use it to generate text completions with the popular Claude LLM family for your existing text data.
 
 # Anthropic
-Anthropic is a AI assitant framework for your usecases. In this handler, we use the Anthropic package, which is available in Python. More information about this Python client can be found (here)[https://github.com/anthropics/anthropic-sdk-python].
+Anthropic is an AI safety and research company. Claude is their assistant model.The package used is the Anthropic Python SDK.More information about this Python client can be found (here)[https://github.com/anthropics/anthropic-sdk-python].
+
+*Note:* For more information on read the (here) [https://docs.anthropic.com/claude/reference/getting-started-with-the-api#evaluation--going-live-with-the-api]
```

Comment:
The syntax on both links is backwards. Markdown supports links with hyperlinked text in square brackets and the link in round brackets, [like so](https://mindsdb.com).